.github/workflows/node.yml

```diff
@@ -63,6 +64,13 @@ jobs:
       - name: Build packages
         run: npm run build
 
+      - name: Add, commit and push changes to dist/generateContents.js
+        if: github.event_name == 'push'
```

Comment:
My bad in fact you could comment or remove the if condition. I thought that this workflow was also catching release.